src/renderer/state/route/test/logics.test.ts

```diff
@@ -337,7 +338,21 @@ describe("Route logics", () => {
         VIEW_UPLOADS_SUCCEEDED
       );
     });
+    it("allows users to open an upload with unexposed annotations", async () => {
+      stubMethods();
```

Comment:
Note: this was `stubMethods({});` in the [original PR](https://github.com/aics-int/aics-file-upload-app/pull/34/files#), but I'm guessing is correct as `stubMethods();`